docs/docs/sample-scripts/spo/list-all-files-with-missing-required-metadata/index.md

```diff
@@ -14,13 +14,13 @@ The following script iterates through all files in all the document libraries of
 
     ```powershell
     $siteUrl = "https://<TENANT-NAME>.sharepoint.com/sites/<YOUR-SITE>"
-    $allLists = m365 spo list list --webUrl $($siteURL) --output json | ConvertFrom-Json
+    $allLists = m365 spo list list --webUrl "$($siteURL)" --output json | ConvertFrom-Json
     $allLibs = $allLists | where {$_.BaseTemplate -eq 101}
     $results = @()
     $fields = @('FileLeafRef', 'FileSystemObjectType')
 
     foreach ($lib in $allLibs) {
-        $allRequiredFields = m365 spo field list --webUrl $($siteURL) --listId $($lib.Id) --query "[?Required == ``true``]" --output json | ConvertFrom-Json
+        $allRequiredFields = m365 spo field list --webUrl "$($siteURL)" --listId $($lib.Id) --query "[?Required == ``true``]" --output json | ConvertFrom-Json
```

Comment:
same comment I think it will still work without the `$()`

```suggestion
        $allRequiredFields = m365 spo field list --webUrl "$siteURL" --listId $($lib.Id) --query "[?Required == ``true``]" --output json | ConvertFrom-Json
```